Studio/src/Application/Optimize/OptimizeTool.ui

```diff
@@ -572,10 +541,10 @@ background: rgb(255, 255, 127)
          </size>
         </property>
         <property name="decimals">
-         <number>1</number>
+         <number>2</number>
         </property>
         <property name="minimum">
-         <double>0.100000000000000</double>
+         <double>0.010000000000000</double>
```

Comment:
not too familiar with regularization but shouldn't we able to set this to 0, in case we want to turn off regularization?

Reply:
Yep, seems like it works to set it to 0.

To disable, I believe you just set starting and ending to the same value.